src/main/java/io/cdap/plugin/gcp/bigquery/sqlengine/BigQueryReadDataset.java

```diff
@@ -75,7 +79,7 @@ public class BigQueryReadDataset implements SQLDataset, BigQuerySQLDataset {
   public static final String SQL_INPUT_CONFIG = "config";
   public static final String SQL_INPUT_FIELDS = "fields";
   public static final String SQL_INPUT_SCHEMA = "schema";
-  private static final Type LIST_OF_STRINGS_TYPE = new TypeToken<ArrayList<String>>() {
+  private static final java.lang.reflect.Type LIST_OF_STRINGS_TYPE = new TypeToken<ArrayList<String>>() {
```

Comment:
I don't think  this fully qualified import is needed. There are no conflicting class names.